dask_kubernetes/operator/kubecluster/kubecluster.py

```diff
@@ -115,6 +115,9 @@ class KubeCluster(Cluster):
         used to create the cluster instead of generating one from the other keyword arguments.
     scheduler_forward_port: int (optional)
         The port to use when forwarding the scheduler dashboard. Will utilize a random port by default
+    quiet: bool
+        If enabled, suppress rich console output.
```

Comment:
I think we should be clear that this suppresses all printed output from the cluster, not just the rich console. I acknowledge that the only output we have today is the rich console but that may not always be true.